infra/resources/modules/commons/function_async.tf

```diff
@@ -16,10 +16,14 @@ locals {
     FETCH_KEEPALIVE_FREE_SOCKET_TIMEOUT = "30000"
     FETCH_KEEPALIVE_TIMEOUT             = "60000"
 
-    COSMOSDB_ENDPOINT      = module.cosmosdb_account.endpoint
-    COSMOSDB_DATABASE_NAME = module.cosmosdb_sql_database_trial.name
-    EVENTHUB_NAMESPACE     = "${module.event_hub.name}.servicebus.windows.net"
-    SERVICEBUS_NAMESPACE   = "${local.servicebus_namespace}.servicebus.windows.net"
+    COSMOSDB_ENDPOINT                 = module.cosmosdb_account.endpoint
+    COSMOSDB_DATABASE_NAME            = module.cosmosdb_sql_database_trial.name
+    EVENTHUB_NAMESPACE                = module.event_hub.name
+    SERVICEBUS_NAMESPACE              = local.servicebus_namespace
+    "SERVICE_BUS_RESOURCE_GROUP_NAME" = azurerm_servicebus_namespace.main.resource_group_name
+    "SERVICE_BUS_LOCATION"            = azurerm_servicebus_namespace.main.location
+
+    "SUBSCRIPTION_ID" = data.azurerm_subscription.current.subscription_id
```

Comment:
**chore**: @Krusty93 is this the right way to get the `PROD-TRIAL` subscription id?

I need the subscriptionId because a process will create resources on Azure when creating a new Trial.

Reply:
In this way you're getting the id of the subscription you're currently logged into (i.e. az cli or GitHub action).
Is this the desired behavior?

Reply:
Ok, nope. I just need the subscription id of the `PROD-TRIAL` subscription, which is the subscription where the resources are going to be created.

At the moment, is there a way to get that value from Terraform?

Reply:
I'm gonna resolve this conversation since we talked about this and it's ok the way we get the `PROD-TRIAL` subscription id